Sources/ApolloCodegenLib/IR/IR+Schema.swift

```diff
@@ -60,3 +65,17 @@ extension IR {
     }
   }
 }
+
+extension GraphQLScalarType {
+  var isCustom: Bool {
+    if let _ = self.specifiedByURL { return true }
+
+    return !isString && !isInt && !isBool && !isFloat && !isID
+  }
+
+  var isString: Bool { self.name == String(describing: String.self) }
+  var isInt: Bool { self.name == String(describing: Int.self) }
+  var isBool: Bool { self.name == "Boolean" }
+  var isFloat: Bool { self.name == String(describing: Float.self) }
+  var isID: Bool { self.name == "ID" }
```

Comment:
`ID` comes through as a scalar type, had to be checking for it too.

Reply:
This seems like a lot of extra work. This could just be

```swift
switch name {
case "String", "Int", "Float", "Boolean", "ID":
  return false
default: 
  return true
}
```